<launch> 
 
   <!-- motor node -->
  <include file="$(find xycar_motor)/launch/xycar_motor.launch" />
  
  <!-- USB Camera node -->
  <include file="$(find usb_cam)/launch/xycar_cam.launch" />


  <node name="start_node" pkg="track_drive" type="start.py" output="screen" />

</launch>



